src/components/map/ThematicLayer.js

```diff
@@ -34,11 +35,9 @@ class ThematicLayer extends Layer {
         this.layer.on('click', this.onFeatureClick, this);
         this.layer.on('contextmenu', this.onFeatureRightClick, this);
 
-        const layerBounds = this.layer.getBounds();
-
-        if (layerBounds.isValid()) {
-            map.invalidateSize();
```

Comment:
Is `map.invalidateSize()` no longer necessary?

Reply:
Yes, I did some tests, and it seems that it's not needed - and it's not used for the other layer types. map.invalidateSize() should be used when the size of the map div is changed, which is not the case here.